lib/game_data.js

```diff
@@ -105,6 +105,14 @@ class GameData {
     */
     addAdditionalItemProperties(iteminfo) {
         if (!this.items_game || !this.items_game_cdn || !this.csgo_english) return;
+        
+        //get sticker codename/name
+        for(var i=0;i<iteminfo.stickers.length;i++){
+            if (iteminfo.stickers[i].sticker_id in this.items_game['items_game']['sticker_kits']) {
+                iteminfo.stickers[i]['codename'] = this.items_game['items_game']['sticker_kits'][iteminfo.stickers[i].sticker_id]['name'];
```

Comment:
This can be refactored into:

```
let sticker_kit = sticker_kits[sticker.sticker_id]

if (sticker_kit) {
    sticker.codename = sticker_kit.name;

    let name = this.csgo_english['lang']['Tokens'][sticker_kit.item_name.replace('#', '')];
    if (name) sticker.name = name;
}
```

We have to account for when the game files haven't been updated and a user requests a new sticker.